src/components/specific/visa/visa-selection-validator/visa-selection-validator-people/VisaSelectionValidatorPeople.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div
+    class="visa-add-validator-people"
+    v-for="people in userList"
```

Comment:
```suggestion
    v-for="user in userList"
```
We usually use `user`